<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="TYSS-Lini-B-Asok" failures="1" tests="2" name="Test" time="3.279" errors="0" timestamp="2023-12-15T23:07:57 IST">
  <testcase classname="My_practice2.ExtendReporterTest" name="Login" time="3.162">
    <failure type="java.lang.AssertionError" message="nvigationfailed expected [false] but found [https://demowebshop.tricentis.com/register]">
      <![CDATA[java.lang.AssertionError: nvigationfailed expected [false] but found [https://demowebshop.tricentis.com/register]
at My_practice2.ExtendReporterTest.Login(ExtendReporterTest.java:38)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:829)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- Login -->
  <testcase classname="My_practice2.HardAssert" name="login" time="3.227"/>
</testsuite> <!-- Test -->
